client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -0,0 +1,160 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { last, isEqual } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { deleteStoredKeyringConnection } from 'state/sharing/keyring/actions';
+import { SharingService, connectFor } from 'my-sites/sharing/connections/service';
+
+export class GoogleMyBusiness extends SharingService {
+	static propTypes = {
+		...SharingService.propTypes,
+		deleteStoredKeyringConnection: PropTypes.func,
+	};
+
+	static defaultProps = {
+		...SharingService.defaultProps,
+		deleteStoredKeyringConnection: () => {},
+	};
+
+	createOrUpdateConnection = () => {};
+
+	/**
+	 * Deletes the passed connections.
+	 *
+	 * @param {Array} connections Optional. Connections to be deleted.
+	 *                            Default: All connections for this service.
+	 */
+	removeConnection = () => {
+		this.setState( { isDisconnecting: true } );
+		this.props.deleteStoredKeyringConnection( last( this.props.keyringConnections ) );
```

Comment:
We don't but it doesn't matter much which one is removed. As long as there is one working connection, it will be marked as connected and thus will propose you to disconnect.

In any case, for the same reason, this component should not allow you to add multiple connections. It may happen if for instance you load the popup from 2 devices (or browser or tabs) simultaneously and connect 2 different accounts but it's quite an edge case imo. We should consider that this service only handles one connection.